### DISCUSSION TITLE
Mailchimp Settings not Saving as Expected

### DISCUSSION THREAD

**Discussion by Nathan269** [statamic/cms#4796] [Troubleshooting]
I'm having a slight issue with my control panel using Statamic. If I make a change to my settings such as changing a toggle from on to off, it says that it was saved successfully. If I then refresh the page it reverts back to what it was before (on). If I then refresh the page again it then changes back to what I wanted it to be (off). The changes are being made successfully, however, I feel like this is a cacheing issue and when I refresh the page, it is just displaying the previous view. Although I'm able to change the settings successfully, this is not ideal and may cause confusion. Any help would be appreciated as to what yous may think is causing this issue.

**Comment by Nathan269** [comment]
This isn't a concerning issue anymore. It's just a slight delay for the changes to be processed. If I wait about 5 seconds or so and refresh the page, the changes should remain the same as what I set them to be.

**Comment by jasonvarga** [comment]
If it's only an issue with Mailchimp I'd bring it up with the addon developer.
